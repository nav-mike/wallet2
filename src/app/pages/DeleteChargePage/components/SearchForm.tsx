import { Select } from 'app/components/Select';
import { SubmitButton } from 'app/components/SubmitButton';
import React from 'react';
import { Form } from './Form';
import { MonthsSelect } from './MonthsSelect';

export const SearchForm = () => {
  return (
    <Form>
      <MonthsSelect>
        <option>January</option>
        <option>February</option>
        <option>December</option>
      </MonthsSelect>
      <Select>
        <option>2020</option>
        <option>2021</option>
      </Select>
      <SubmitButton>Search</SubmitButton>
    </Form>
  );
};
