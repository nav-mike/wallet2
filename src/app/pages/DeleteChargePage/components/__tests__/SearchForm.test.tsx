import * as React from 'react';
import { render } from '@testing-library/react';
import { SearchForm } from '../SearchForm';

describe('<SearchForm />', () => {
  it('should render and match the snapshot', () => {
    const form = render(<SearchForm />);
    expect(form.container.firstChild).toMatchSnapshot();
  });
});
