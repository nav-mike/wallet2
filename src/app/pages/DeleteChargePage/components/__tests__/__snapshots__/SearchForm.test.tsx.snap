// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchForm /> should render and match the snapshot 1`] = `
.c2 {
  border: 1px solid rgba(220,120,95,1);
  border-radius: 0.3rem;
  padding: 0.2rem;
}

.c2:hover {
  cursor: pointer;
}

.c3 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  background-color: rgba(220,120,95,1);
  border: 1px solid rgba(220,120,95,1);
  border-radius: 0.5rem;
  min-height: 2rem;
}

.c3:hover {
  cursor: pointer;
  background-color: #fab9a9;
}

.c0 {
  margin: 1rem 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c1 {
  border: 1px solid rgba(220,120,95,1);
  border-radius: 0.3rem;
  padding: 0.2rem;
  -webkit-flex: 2;
  -ms-flex: 2;
  flex: 2;
}

.c1:hover {
  cursor: pointer;
}

<form
  class="c0"
>
  <select
    class="c1"
  >
    <option>
      January
    </option>
    <option>
      February
    </option>
    <option>
      December
    </option>
  </select>
  <select
    class="c2"
  >
    <option>
      2020
    </option>
    <option>
      2021
    </option>
  </select>
  <button
    class="c3"
  >
    Search
  </button>
</form>
`;
